ci/images/plugin-compiler/data/build.sh

```diff
@@ -64,7 +44,7 @@ yes | cp -rf $PLUGIN_BUILD_PATH/vendor/* $GOPATH/src || true \
 # First we need to find which deps GW already has, remove this folders, and after copy fresh versions from GW
 
 # github.com and rest of packages have different nesting levels, so have to handle it separately
-ls -d $TYK_GW_PATH/vendor/github.com/*/* | sed "s|$TYK_GW_PATH/vendor|$GOPATH/src|g" | xargs -d '\n' rm -rf
+ls -d $TYK_GW_PATH/vendor/github.com/*/* | sed "s|$TYK_GW_PATH/vendor|$GOPATH/src|g" | grep -v "TykTechnologies/tyk" | xargs -d '\n' rm -rf
```

Comment:
I wonder why we have this line here. It limits this fix only to gateway packages. What was the original commit which changed exact this line? I wonder if we should keep this bugfix.

Reply:
Yes, we can remove that. It was added to fix the issue with the `certs` package being included as a submodule in go.mod previously - in this line, that led to the gateway source being deleted from the GOPATH. Since this was removed later on lts, we can change this back to the older line.

Reply:
Yes lets do